docs/skopeo-sync.1.md

```diff
@@ -0,0 +1,80 @@
+% skopeo-sync(1)
+
+## NAME
+skopeo\-sync - Copy images from one or more repositories to a user specified destination.
+
+## SYNOPSIS
+**skopeo sync** [**--source-yaml**] _source_ _destination_
+
+## DESCRIPTION
+Copy images from one or more repositories to a user specified destination.
```

Comment:
Please give some more details about the internals. Which images are copied? Will they be updated if the destination has an image with the same name already? Is there something like path traversal/walks?

Reply:
Starting with a single sentence is good; most of this (like the “path traversal/walk” question) is better placed in the description of individual source/destination formats.

“What does ‘sync’ mean” should be explained somewhere around here, though.